packages/ember-headless-form/src/components/headless-form.ts

```diff
@@ -62,6 +62,13 @@ export interface HeadlessFormComponentSignature<
      */
     validate?: FormValidateCallback<DATA>;
 
+    /**
+     * Allows you to opt-out of native validation.
+     *
+     * This can be useful if all of the validation logic is already handled by the `@validate` hooks, but you have form controls that have validation requirements (e.g. `email` type) that would cause the native validation to interfere or.
+     */
+    ignoreNativeValidation?: boolean;
```

Comment:
A bit of an open question: now that we are adding more `validation`-related arguments, should we consider having a `validation` object that's a grab-bag of all the validation arguments, something like:

```
validation: {
  validateOn: ...
  revalidateOn: ...
  ignoreNativeValidation: ...
  validate: ... 
}
```